samples/addons/kmesh_performance-monitoring.json

```diff
@@ -0,0 +1,695 @@
+{
```

Comment:
These dashbords should be placed into https://github.com/kmesh-net/kmesh/blob/main/samples/addons/grafana.yaml, so we can see it through grafana, ref: https://kmesh.net/en/docs/userguide/service_metrics/ , please also update the doc in this website.